### ISSUE TITLE
Can we please have the ctrl mouse wheel change the code font 

### ISSUE THREAD

**Issue by icuurd12b42** [Microsoft/vscode#34890]
- VSCode Version: Code 1.16.1 (27492b6bf3acb0775d82d2f87b25a93490673c6d, 2017-09-14T16:38:23.027Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-autohotkey|sle|0.2.1

I hate it when the standards are not respected, I'm sure you noticed with all my rantings.

My issue of the day is regarding the Code Editor Font Size not attached the ctrl+mouse wheel like every text/code editor out there. There's a reason the change font size is attached to the mouse wheel. people use it, especially if like me they are half blind, we use it hundreds of times an hour

You absolutely need the ability to change the font size on the fly in the code editor window because this is how blind people code:

https://www.youtube.com/watch?v=RqGUhLW4pbs

It's not the first time I notice this omission in a code a editor, I already made that video for another Code editor whose developer also made the same bad judgement call. It's understandable if you are not visually impaired but this needs addressing.

I know you can change the UI font with ctrl - or + but that is NOT what I'm asking. for one it changes the wrong thing and for 2 it breaks the fluidity of the perusing. 

Like in Code, here, the people I made the video for also thought ctlr - or + to change the UI size was what was people needed... it's not. We need both UI AND Code size change to support people with poor eyesight. 

And I'm NOT asking to tie the mouse wheel to the UI font Size!!! It needs to be tied to the Code Box ONLY!

I dont want to zoom in my code and the whole UI explodes leaving me with no room to work with,

ctrl - or + to change the UI zoom
ctrl mouse wheel to change the codebox zoom
the code box font size is determined by the code box font settings * the codebox zoom * the UI zoom

Anyway, I hope you guys fix it.... And remember, I would not be posting suggestions or pointing out omissions if I did not care. I'm starting to like this product a whole lot tbh, but these little annoyances take their toll and end up being major hurdles over long periods of time, they need to be fixed. 

---

Steps to Reproduce:

1.
2.

Reproduces without extensions: Yes/No

**Comment by yume-chan** [comment]
If I understand correctly, you want to use <kbd>Ctrl</kbd> + <kbd>Mouse Wheel</kbd> to change the editor font size, but not the UI font size? Then you can set the `editor.mouseWheelZoom` setting to `true`.

If you want to use keyboard shortcuts to change the editor font size, you can try the [Editor Zoom](https://marketplace.visualstudio.com/items?itemName=zolocode.editorzoom) extension.

**Comment by alexandrudima** [comment]
@icuurd12b42 The setting is `editor.mouseWheelZoom`, just like @yume-chan mentions. 

#### The setting is off by default because:
Our challenge is that we are built on top of Electron, which is built on top of Chromium, so by the time we receive a `mousewheel` event, we no longer know if this came from a physical mouse wheel or if it came in from a TouchPad. 
Since we cannot discriminate between the two sources, every slight touch (even by accident) of the TouchPad when holding `ctrl` (which happens often for folks using a laptop) will trigger the font size change.

**Comment by icuurd12b42** [comment]
"editor.mouseWheelZoom": true 

does fix it. 

You guys thought of everything, but I would debate the wiseness of your decisions on what the out the box default settings.are, especially on that one....

...frankly I would think most users use code on a PC with a mouse and KB; it's rather odd to choose erratic behaviour for most in favor of a the rare few that use a touchpad, who, quite frankly, can resolve their issue the same way I did, by turning it off. Let them come here and complain ;). The touch pads I used, the last 3 pcs I own were laptops, had a dedicated wheel zone on the pad which I used for the same zoom in and out when coding... So I'm not exactly sure of the issue there... But hey I'm not neck deep in your code.
